dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2295,6 +2313,98 @@
         }
       }
     },
+    "ZeroShotClassificationTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for a Zero Shot Classification task.",
+      "properties": {
+        "classificationType": {
+          "type": "string",
+          "description": "Specifies either one or multiple categories per document. Defaults to multi classification which may return more than one class for each document.",
+          "default": "Multi",
+          "enum": [
+            "Single",
+            "Multi"
+          ],
+          "x-ms-enum": {
+            "name": "ClassificationType",
+            "modelAsString": true
+          }
+        },
+        "categories" : {
+          "type": "array",
+          "description": "a list of categories to which input is classified to.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "allOf": [
+          {
+            "$ref": "#/definitions/TaskParameters"
+          }
+        ],
+        "required": [
+          "categories"
+        ]
+      }
+    },
+    "ZeroShotClassificationTaskResult": {
+      "type": "object",
+      "properties": {
+        "results": {
+          "$ref": "#/definitions/ZeroShotClassificationResult"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextTaskResult"
+        }
+      ],
+      "required": [
+        "results"
+      ],
+      "x-ms-discriminator-value": "ZeroShotClassificationResults"
+    },
+    "ZeroShotClassificationResult": {
+      "type": "object",
+      "properties": {
+        "documents": {
+          "type": "array",
+          "description": "Response by document",
+          "items": {
+            "allOf": [
+              {
+                "$ref": "#/definitions/ClassificationDocumentResult"
+              }
+            ]
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ZeroShotResult"
+        }
+      ],
+      "required": [
+        "documents"
+      ]
+    },
+    "ZeroShotResult": {
+      "properties": {
```

Comment:
is there no modelVersion associated with this task?

Reply:
good point actually no there is no model version. The model version is to be retried for prebuilt so I did not want to add it for this capability from the beginning. I am planning to bring this up in the board review as well to discuss

Reply:
Discussed with the team and we agreed to keep the model version as the rest of prebuilts capabilities and retire it with the rest of prebuilts. to avoid forking in the result